polaris-core/src/main/java/org/apache/polaris/core/persistence/pagination/EntityIdPaging.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.core.persistence.pagination;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+import jakarta.annotation.Nullable;
+import org.apache.polaris.core.entity.PolarisBaseEntity;
+
+/** Utility class for processing pagination of streams of entities ordered by some ID property. */
+public final class EntityIdPaging {
+  private EntityIdPaging() {}
+
+  /**
+   * Produces the reference to the next page of data in the form of a numerical entity ID. Entities
+   * in the associated stream of data are assumed to be ordered by ID.
```

Comment:
What stream? This is just taking one entity and returning `Long.toString(entity.getId())`.

Reply:
This is in reference to how the document method is used in relation to handling paginated requests and responses (which deal with streams of data).